pandas/tests/io/formats/test_printing.py

```diff
@@ -218,11 +218,23 @@ def test_multiindex_long_element():
         ([-2, complex("nan"), -1], ["-2.0+0.0j", " NaN+0.0j", "-1.0+0.0j"]),
         ([-1.23j, complex("nan"), -1], ["-0.00-1.23j", "  NaN+0.00j", "-1.00+0.00j"]),
         ([1.23j, complex("nan"), 1.23], [" 0.00+1.23j", "  NaN+0.00j", " 1.23+0.00j"]),
+        (
+            [-1.23j, complex(np.nan, np.nan), 1],
+            ["-0.00-1.23j", "  NaN+ NaNj", " 1.00+0.00j"],
```

Comment:
Is there supposed to be a space between `+ NaN`?

```
In [5]: complex(1.2, float("nan"))
Out[5]: (1.2+nanj)

In [6]: complex(float("nan"), float("nan"))
Out[6]: (nan+nanj)
```

Reply:
I'm trying to align +/-. For instance,
```
0   1.23456+1.23456j
1       NaN+    NaNj
```
Not sure if there is a better way to display this.

Reply:
Ah good call. Yeah this if fine then